Platform/AlderlakeBoardPkg/Script/StitchIfwiConfig_adl.py

```diff
@@ -0,0 +1,317 @@
+## @ StitchIfwiConfig.py
```

Comment:
Are you going to use a single stitch config file to support different ADL SKUs? If yes,  Maybe rename filename to StitchIfwiConfig.py to align with above comments. If no, better rename it to StitchIfwiConfig_adls.py and update the comments above.